chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/CertificationEditionDAOImpl.java

```diff
@@ -18,6 +19,7 @@
 public class CertificationEditionDAOImpl extends BaseDAOImpl implements CertificationEditionDAO {
 
     @Override
+    @Transactional
```

Comment:
Why are all of these transactional? Is this as a result of the need to do the "delete and save" inside a single transaction? If so, these methods probably don't need this annotation